stdlib/copy.pyi

```diff
@@ -1,8 +1,15 @@
-from typing import Any, TypeVar
+import sys
+from typing import Any, Protocol, TypeVar
+from typing_extensions import Self
 
 __all__ = ["Error", "copy", "deepcopy"]
 
 _T = TypeVar("_T")
+_SR = TypeVar("_SR", bound=_SupportsReplace)
+
+class _SupportsReplace(Protocol):
+    # In reality doesn't support args, but there's no other great way to express this.
```

Comment:
Technically speaking, it does support positionals, but those will be passed as keywords. What it does not support however is positional-only arguments.

On the other hand, you can just force people to use keyword-only arguments so that it matches https://docs.python.org/3.13/library/copy.html#object.__replace__. Or at least, make `self` a positional-only argument.

Reply:
>>> On the other hand, you can just force people to use keyword-only arguments so that it matches

The only issue is that won't work that way you might think with either MyPy or Pyright. Take this example: https://pyright-play.net/?pythonVersion=3.13&strict=true&code=GYJw9gtgBALgngBwJYDsDmUkQWEMoAK4MYAxmADYA0UAginAFDOkUCGAzh1APoDKAVwQ48HAEoBTBO1ISAFETAlyFAJQAuRlG1QAJhOC8eIKTIk8ecjhIrAaAKnsBrAO5sQaDuroNVUALQAfFAAcmAoEt4AdDHMjKyc3ACS2BSaOnoGRibSbLIWVjZ2UO5oPACM3qgwfkGh4ZFQMVHM%2BoYwEhwwcjkybABGFI38QiIw4qZ5ErXBYRHRscxY0lAAvFAp0nKqjB1dcstqQA

Reply:
Yes, callables with `*args: Any, **kwargs: Any` are treated specially in the spec.